src/Crdt/ISyncable.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Crdt.Core;
+
+namespace Crdt;
+
+public interface ISyncable
+{
+    Task AddRangeFromSync(IEnumerable<Commit> commits);
+    Task<SyncState> GetSyncState();
+    Task<ChangesResult<Commit>> GetChanges(SyncState otherHeads);
+    Task<SyncResults> SyncWith(ISyncable remoteModel);
+    Task SyncMany(ISyncable[] remotes);
+    ValueTask<bool> ShouldSync();
+}
+
+public class NullSyncable : ISyncable
```

Comment:
yes, though there might be a better way to deal with it. In FLEx lite, it's used when working on a local only project, rather than not calling sync on a project that's local only we just call sync and use a nullSyncable as the remote. We could move this into FLEx lite, but it seemed like it might be useful else where.